lib/rubocop/cop/style/unless_with_operator.rb

```diff
@@ -0,0 +1,28 @@
+# frozen_string_literal: true
+
+# TODO: when finished, run `rake generate_cops_documentation` to update the docs
+module RuboCop
+  module Cop
+    module Style
+      # This cop looks for *unless* with conditionals including operators.
+      #
+      # @example
+      #   # bad
+      #   return false unless foo && bar
+      #
+      #   # good
+      #   return false unless foo
+      #   return false unless bar
+      class UnlessWithOperator < Cop
+        MSG = 'Do not use `unless` with operator. Split this in several lines.'
+
+        def on_if(node)
+          return unless node.unless?
+          return unless node.condition.respond_to?(:operator)
```

Comment:
Practice what you preach 😄

Reply:
How about moving the condition to an intent explicit method?
```rb
def on_if(node)
  add_offense(node) if unless_with_operator?(node)
end

private

def unless_with_operator?(node)
  node.unless? && node.condition.respond_to?(:operator)
end
```